pkg/apis/eventing/v1alpha1/channel_types.go

```diff
@@ -80,24 +87,12 @@ type ChannelSpec struct {
 	// +optional
 	Arguments *runtime.RawExtension `json:"arguments,omitempty"`
 
-	// Subscribers is a list of the Subscribers to this channel. This is filled in
-	// by the Subscriptions controller. Users should not mutate this field.
-	Subscribers []ChannelSubscriberSpec `json:"subscribers,omitempty"`
+	Channelable *Channelable `json:"channelable,omitempty"`
 }
 
-// ChannelSubscriberSpec defines a single subscriber to a Channel. At least one
-// of Call or Result must be present.
+// ChannelSubscriberSpec defines a single subscriber to a Channel.
 type ChannelSubscriberSpec struct {
-	// Call is an optional reference to a function for processing events.
-	// Events from the From channel will be delivered here and replies
-	// are optionally handled by Result.
-	// +optional
-	Call *Callable `json:"call,omitempty"`
-
-	// Result optionally specifies how to handle events received from the Call
-	// target.
-	// +optional
-	Result *ResultStrategy `json:"result,omitempty"`
+	Sinkable string `json:"sinkable"`
```

Comment:
I thought the dispatcher acts upon the channel's resolved subscriptions to determine the target DNS [1] to send an event and optionally handle replies depending on whether the target was a `Callable`. With this change to a single `Sinkable`, I am failing to understand how the reply logic will be handled by the dispatcher? 

[[1]](https://docs.google.com/presentation/d/1bUAqTILr2ElQkdQ8to2BrUDC0O7Cja9lE6kGMDmEPzA/edit#slide=id.g4259f7357d_0_34)

Reply:
Yes, that's exactly right. But this is really for connecting Subscriptions together, well, really Sources to Channels. Say I have the following:
 Source -> Call -> Result
Result is really a channel and if another subscription wants to connect to that channel, the model will look more like this:
Source -> Call -> Result (Now becomes a Source2) -> Call -> Result2
But, we need to be able to support multiple subscriptions to the channel, so it really looks more like this:
Source -> Call -> Result -> SourceChannel -> Call -> Result2
                                         \-> SourceChannel2 -> Call -> Result3

In this model, the "Result" Channel will only get SInkable, since they represent "input (or Source)" channels fronting Subscriptions and will allow for Fanout. We do not want the Dispatcher to be handling failures in fanouts.

Does that make more sense?